src/qml/QFieldCloudScreen.qml

```diff
@@ -583,6 +589,39 @@ Page {
     }
   }
 
+  QfMenu {
+    id: options
+    x: parent.width - width - 8
+
+    MenuItem {
+      text: qsTr('Show invalid projects')
+      font: Theme.defaultFont
+      height: 48
+      leftPadding: Theme.menuItemLeftPadding
+      checkable: true
+      checked: settings ? settings.valueBool("/QField/showInvalidProjects", false) : false
+      onTriggered: {
+        settings.setValue("/QField/showInvalidProjects", checked);
+        table.model.showInValidProjects = checked;
+      }
+    }
+
+    MenuSeparator {
+    }
+
+    MenuItem {
+      text: qsTr('Sign out')
```

Comment:
Can you move this to sit on top of the show invalid projects, and have it be a sign in action when the app is signed out.

The sign in should open the login panel so people manually hit the sign in button (vs. automated sign out, which logs us out)